### PR TITLE
Add Azure Application Gateway as adopt

### PR DESCRIPTION
**Pull request by Krusty93**
- [X] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [X] I've added links to the entry's documentation and/or
      website
- [X] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [X] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [X] I've described at least one use case for the entry

### CODE REVIEW

File: radar/platforms-and-services/azure-application-gateway.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+### Use cases
+
+Opt for Application Gateway when you need to load balance web applications (HTTP/HTTPS) that can be exposed on internet or not. Remember it is a regional service, so if your application runs in multiple regions you need to replicate the Application Gateway as well (opposed to the [Front Door](https://learn.microsoft.com/en-us/azure/frontdoor/front-door-overview)).
+Application Gateway is often put in front of an [APIM](https://azure.microsoft.com/en-us/products/api-management) instance for its WAF capabilities, DDoS protection and SSL offloading.
```
Comment (gunzip) [inline radar/platforms-and-services/azure-application-gateway.md:13]: It's not clear if we should put it in front of the APIM or if it's a weak suggestion.

File: radar/platforms-and-services/azure-application-gateway.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+You can also use Application Gateway as ingress controller for your AKS cluster.
```
Comment (gunzip) [inline radar/platforms-and-services/azure-application-gateway.md:15]: I won't put any reference to AKS if we're not going to put it in this trechnology radar.

File: radar/platforms-and-services/azure-application-gateway.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+A specific [implementation for containers](https://learn.microsoft.com/en-us/azure/application-gateway/for-containers/overview) is currently in preview.
```
Comment (gunzip) [inline radar/platforms-and-services/azure-application-gateway.md:17]: What do you mean with "for cointainers"?

**Review by gunzip — CHANGES_REQUESTED** [review]

File: radar/platforms-and-services/azure-application-gateway.md (hunk 3 above)
Comment (Krusty93) [reply radar/platforms-and-services/azure-application-gateway.md:17]: It's a new load balancer based on current Application Gateway designed for containers. "For containers" is the product name

File: radar/platforms-and-services/azure-application-gateway.md (hunk 2 above)
Comment (Krusty93) [reply radar/platforms-and-services/azure-application-gateway.md:15]: I think we should add it and we already using the Application Gateway as ingress controller

File: radar/platforms-and-services/azure-application-gateway.md (hunk 1 above)
Comment (Krusty93) [reply radar/platforms-and-services/azure-application-gateway.md:13]: It's a best practice and I added some example: `for its WAF capabilities, DDoS protection and SSL offloading.`
Application Gateway is a load balancer, APIM is not

File: radar/platforms-and-services/azure-application-gateway.md (hunk 2 above)
Comment (gunzip) [reply radar/platforms-and-services/azure-application-gateway.md:15]: I'd proceed to discuss the AKS entry first, and only when in 'adopt' adding these lines here.

File: radar/platforms-and-services/azure-application-gateway.md (hunk 3 above)
Comment (Krusty93) [reply radar/platforms-and-services/azure-application-gateway.md:17]: Yes, but in my opinion it is going to be used also for container apps in the hidden [`MC` resource group](https://serverfault.com/questions/1092550/what-does-the-mc-prefix-come-from-in-aks#:~:text=When%20creating%20a%20cluster%20in,resources%20associated%20with%20the%20cluster.)

Comment (gunzip) [reply radar/platforms-and-services/azure-application-gateway.md:17]: if it's hidden we don't care, correct?

Comment (Krusty93) [reply radar/platforms-and-services/azure-application-gateway.md:17]: Yeah, I don't know whether to keep the reference just to make aware the reader of this new feature to play with

**Review by gunzip — APPROVED** [review]

**Review by uolter — APPROVED** [review]

**Pull request merged**